src/chain_spec.rs

```diff
@@ -620,7 +620,7 @@ fn alcyone_testnet_genesis(
 
     AlcyoneConfig::GenesisConfig {
         frame_system: Some(AlcyoneConfig::SystemConfig {
-            code: alcyone::WASM_BINARY.to_vec(),
+            code: alcyone::WASM_BINARY.unwrap_or_default().to_vec(),
```

Comment:
We should panic here instead of defaulting. We don't want scenarios where we have the release binary without wasm.

Reply:
The thing is this allows us to use `BUILD_DUMMY_WASM_BINARY` on building, which makes faster compilations.
I use it a lot, but I could have this patch locally, so I am quite agnostic to have `unwrap` or `unwrap_or_default`.

WDYT? @maxsam4

Reply:
Let's put this behind a feature flag that we can use when developing but avoid when building production binaries.